### PR TITLE
docs: add deprecation notice for the official conda channel and more conda docs

### PR DESCRIPTION
**Pull request by njzjz**

### CODE REVIEW

File: doc/install/easy-install.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ After your easy installation, DeePMD-kit (`dp`) and LAMMPS (`lmp`) will be avail
 
 :::{note}
 Note: The off-line packages and conda packages require the [GNU C Library](https://www.gnu.org/software/libc/) 2.17 or above. The GPU version requires [compatible NVIDIA driver](https://docs.nvidia.com/deploy/cuda-compatibility/index.html#minor-version-compatibility) to be installed in advance. It is possible to force conda to [override detection](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html#overriding-detected-packages) when installation, but these requirements are still necessary during runtime.
+You can refer to [DeepModeling conda FAQ](https://docs.deepmodeling.com/faq/conda.html) for more information.
```
Comment (njzjz) [inline doc/install/easy-install.md:9]: The link is not ready now. Will finish https://github.com/deepmodeling/docs/pull/21 soon (when I have time).

**Comment by njzjz** [comment]
The CI issue should be fixed by #3461

File: doc/install/easy-install.md [hunk 2]
```diff
@@ -47,27 +66,16 @@ conda create -n deepmd deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps cudatoolkit=11.
 ```
 One could change the CUDA Toolkit version from `10.2` or `11.6`.
```
Comment (caic99) [inline doc/install/easy-install.md:67]: Should CUDA Toolkit versions be changed?

File: doc/install/easy-install.md [hunk 3]
```diff
@@ -47,27 +66,16 @@ conda create -n deepmd deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps cudatoolkit=11.
...
-
-```bash
-conda create -n deepmd deepmd-kit lammps horovod -c conda-forge
-```
-
-The supported platform includes Linux x86-64, macOS x86-64, and macOS arm64.
-Read [conda-forge FAQ](https://conda-forge.org/docs/user/tipsandtricks.html#installing-cuda-enabled-packages-like-tensorflow-and-pytorch) to learn how to install CUDA-enabled packages.
-
 ## Install with docker
 A docker for installing the DeePMD-kit is available [here](https://github.com/orgs/deepmodeling/packages/container/package/deepmd-kit).
```
Comment (caic99) [inline doc/install/easy-install.md:80]: Shall we mention the image from [dockerhub](https://hub.docker.com/r/deepmodeling/deepmd-kit)?

Comment (caic99) [inline doc/install/easy-install.md:80]: Update dead link
```suggestion
A docker for installing the DeePMD-kit is available [here](https://github.com/deepmodeling/deepmd-kit/pkgs/container/deepmd-kit).
```

**Review by caic99 — APPROVED** [review]

File: doc/install/easy-install.md [hunk 4]
```diff
@@ -32,9 +33,27 @@ conda activate /path/to/deepmd-kit
...
-### Official channel
+### conda-forge channel
+
+DeePMD-kit is available on the [conda-forge](https://conda-forge.org/) channel:
+
+```bash
+conda create -n deepmd deepmd-kit lammps horovod -c conda-forge
+```
+
+The supported platform includes Linux x86-64, macOS x86-64, and macOS arm64.
```
Comment (caic99) [inline doc/install/easy-install.md:47]:
```suggestion
The supported platforms include Linux x86-64, macOS x86-64, and macOS arm64.
```

File: doc/install/easy-install.md (hunk 3 above)
Comment (njzjz) [reply doc/install/easy-install.md:82]: It seems not dead but redirects to the new link.

Comment (njzjz) [reply doc/install/easy-install.md:82]: I am not sure whether dockerhub still deletes inactive images: https://www.docker.com/blog/docker-hub-image-retention-policy-delayed-and-subscription-updates/#:~:text=Two%20months%20ago%2C%20we%20announced,after%20six%20months%20of%20inactivity.

Comment (njzjz) [reply doc/install/easy-install.md:82]: Indeed I don't find any images were deleted

File: doc/install/easy-install.md (hunk 2 above)
Comment (njzjz) [reply doc/install/easy-install.md:69]: The CUDA versions in the official channel weren't updated

**Pull request merged**